capella_model_explorer/app.py

```diff
@@ -164,6 +164,20 @@ def rendered_report(template_id: str, model_element_uuid: str = "") -> t.Any:
             cls="prose svg-display dark:prose-invert",
         )
     )
+    return (
+        content,
+        fh.HttpHeader(k="Cache-Control", v="max-age=31536000"),
+        fh.HttpHeader(
+            k="Vary",
+            v=(
+                "capella-model-explorer-version,"
+                "capellambse-version,"
+                "capellambse-context-diagrams-version,"
+                "template-file-hash,"
+                "model-version"
```

Comment:
```suggestion
                "capella-model-explorer-version"
                ",capellambse-version"
                ",capellambse-context-diagrams-version"
                ",template-file-hash"
                ",model-version"
```

This related to the discussion with trailing / leading whitespaces in multi-line strings. The arguments apply here as well.

Reply:
I pushed some change